python/ray/tune/tests/test_api.py

```diff
@@ -1,4 +1,5 @@
 import copy
```

Comment:
Looks like the [lint job](https://buildkite.com/ray-project/ray-builders-pr/builds/32536#819b4d4e-06e5-44cd-a0ca-2e9dddc99ead) failed.

You can run the lint check locally by calling `scripts/format.sh` - further instructions [here](https://docs.ray.io/en/latest/ray-contribute/getting-involved.html#lint-and-formatting)!

Reply:
corrected code format